pydoctor/epydoc2stan.py

```diff
@@ -299,24 +308,25 @@ class FieldHandler:
 
     def __init__(self, obj: model.Documentable):
         self.obj = obj
+        self.linker = _EpydocLinker(self.obj)
```

Comment:
I guess that this is public just to be consistent with the other instance variables ... from the usage, it looks like it could be made private.

Reply:
I don't really give public vs private much thought for pydoctor, since the API is a mess anyway. But I guess we have to start somewhere with cleaning it up.